readme.txt

```diff
@@ -178,6 +178,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Fix - Fixed ticket total formatting when using non-standard thousands and decimal separators. [ET-1197]
```

Comment:
```suggestion
* Fix - Fixed ticket total formatting when using custom thousands and decimal separators. [ET-1197]
```
It's actually standard in Germany to use `,` as a decimal separator ;)

Reply:
Good call and great suggestion!